projects/angular-unleash-proxy-client/src/lib/directives/feature-disabled.directive.ts

```diff
@@ -1,20 +1,39 @@
 import { Directive, Input, TemplateRef } from '@angular/core';
 import { FeatureDirective } from './feature.directive';
-import { NgIfContext } from '@angular/common';
+import { NgIf, NgIfContext } from '@angular/common';
+import { combineLatest, map } from 'rxjs';
+import { JoinOperator } from './join-operator';
 
 @Directive({
   selector: '[featureDisabled]',
   standalone: true,
+  hostDirectives: [NgIf],
 })
 export class FeatureDisabledDirective extends FeatureDirective {
-  @Input('featureDisabled') set toggleName(val: string) {
-    this.setToggleName(val);
+  @Input('featureDisabled') set toggleNames(val: string | string[]) {
+    this.setToggleNames(Array.isArray(val) ? val : [val]);
   }
 
-  @Input('featureDisabledElse')
-  set else(templateRef: TemplateRef<NgIfContext<boolean>> | null) {
-    super.ngIfElse = templateRef;
+  @Input('featureDisabledOperator') set operator(val: JoinOperator) {
+    this._operator = val.toLowerCase() as JoinOperator;
```

Comment:
if the operator is dynamic (changes between 'or' and 'and'), should the state be updated immediately? (without waiting for the toggle names to be updated)

Reply:
I was also thinking about that and decided that no, we shouldn't react on operator changes, it is assumed to be static, but let's discuss and find pros and cons